native_lib/DDGCmdline.c

```diff
@@ -0,0 +1,47 @@
+#include <stdio.h>
+#include <string.h>
+#include <sodium.h>
+#include "DDGSyncCrypto.h"
+
+int main(int argc, char **argv) {
+    if (argc != 3) {
+        printf("Need 2 args, got %d\n", argc);
```

Comment:
can you clarify what the arguments are supposed to be? I'm looking at the code, and I can't figure it out.

Reply:
yes, I made a more standard usage message.  For the record they are "username" and "password" in this version.